assignment/functions/utils.py

```diff
@@ -1,3 +1,67 @@
+import glob
+import pandas as pd
+import numpy as np
+
 # Utility functions for processing .csv data in assignment 6
+def get_files(path_pattern):
+    return glob.glob(path_pattern)
+
 
 # get_files function:
+def get_files1(path_pattern):
+    return glob.glob(path_pattern)
```

Comment:
Why is there a duplicate here? Also this is not the fully functional `get_files` function, this won't sort your files in alphanumerical order. 

-5%